lib/scr/my_app.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+import 'package:simple_dice_roller/scr/screen_dice_roller.dart';
```

Comment:
Change for a relative import here, please

Reply:
Done